daemon/daemon.go

```diff
@@ -237,9 +238,15 @@ func (d *Daemon) compileBase() error {
 		log.Warningf("Command output:\n%s", out)
 		return err
 	}
+
 	log.Warningf("Enabled bpf_jit_enable")
 	log.Warningf("Disabled rp_filter on all interfaces!")
 
+	fout, err := ioutil.ReadFile(filepath.Join(d.conf.RunDir, "bpf_features.log"))
```

Comment:
I see this here but when is this (`bpf_features.log`) file created? And by whom?